src/styles/card.scss

```diff
@@ -0,0 +1,98 @@
+html {
+  font-family: Roboto, Helvetica, sans-serif;
+  font-size: 12px;
+  line-height: 18px;
+}
+
+body {
+  margin: 0;
+}
+
+.card {
+  max-width: $card-width;
+  width: 100%;
+  box-sizing: border-box;
+  border: 1px solid white;
+  padding: 32px 16px 16px;
+
+  &__img {
+    display: block;
+    margin: 0 auto 40px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__title {
+    font-size: inherit;
+    margin: 0;
+    margin-bottom: 4px;
+    font-weight: 500;
+    color: $black-color;
+  }
+
+  &__product-code {
+    margin: 0;
+    margin-bottom: 16px;
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: $grey-color;
+  }
+
+  &__reviews {
+    display: flex;
+    justify-content: space-between;
+    font-size: 10px;
+    margin: 0;
+    margin-bottom: 24px;
+    align-items: center;
+  }
+
+  &__count-review {
+    margin: 0;
+    line-height: 16px;
+  }
+
+  &__price-container {
+    display: flex;
+    justify-content: space-between;
+    margin: 0;
+    margin-bottom: 16px;
+  }
+
+  &__price-text {
+    font-size: 12px;
+    align-items: center;
+    color: $grey-color;
+  }
+
+  &__price-value {
+    font-size: 16px;
+    font-weight: 700;
+  }
+
+  &__button-submit {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    box-sizing: border-box;
+    width: 100%;
+    height: 40px;
+    font-size: 14px;
```

Comment:
organize your CSS
I recommend you take a look at these style codes and use it in the feature
[9elements](https://9elements.com/css-rule-order) or [htmlAcademy](https://codeguide.academy/html-css.html#css-order)